modules/dnn_superres/src/dnn_superres_quality.cpp

```diff
@@ -0,0 +1,470 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#include "precomp.hpp"
+
+#include "opencv2/dnn_superres_quality.hpp"
+
+namespace cv
+{
+    namespace dnn_superres
+    {
+        int DnnSuperResQuality::fontFace = cv::FONT_HERSHEY_COMPLEX_SMALL;
+
+        double DnnSuperResQuality::fontScale = 1.0;
+
+        Scalar DnnSuperResQuality::fontColor = Scalar(255,255,255);
+
+        void DnnSuperResQuality::setFontFace(int fontface)
+        {
+            fontFace = fontface;
+        }
+
+        void DnnSuperResQuality::setFontScale(double fontscale)
+        {
+            fontScale = fontscale;
+        }
+
+        void DnnSuperResQuality::setFontColor(cv::Scalar fontcolor)
+        {
+            fontColor = fontcolor;
+        }
+
+        double DnnSuperResQuality::psnr(Mat img, Mat orig)
```

Comment:
Please use `PSNR()` function from core module instead

Reply:
Replaced it with core module function.